scripts/generate-react-exports.js

```diff
@@ -73,33 +73,29 @@ for (const module of customElementsModules) {
               event.type.text
             )}>`
         )
-        .join(', ')}}`
-    : '';
+        .join(", ")}}`
+    : "";
 
-  const importPath = path.replace(/^src\//, '').replace(/\.ts$/, '');
-  const Type = componentName + 'Type';
+  const importPath = path.replace(/^src\//, "").replace(/\.ts$/, "");
+  const Type = componentName + "Type";
+  const componentType = `${Type}${eventTypes}`;
 
   importStatements.push(`import type ${Type} from "./${importPath}";`);
   exportStatements.push(`export declare type ${componentName} = ${Type}`);
 
-  const componentDefinition =
-    typeParam => `  customElements.whenDefined('${fileName}').then(() => ({
-    default: createComponent${typeParam}(
-      {
-        react: React,
-        tagName: '${fileName}',
-        elementClass: customElements.get('${fileName}'),
-        events: ${JSON.stringify(eventNames)}
-      }
-    )
-  }))`;
-
-  const componentType = `<${Type}${eventTypes}>`;
-  const componentPromise = `() => ${componentDefinition(componentType)}`;
-  const componentLazy = `React.lazy<ReactWebComponent${componentType}>(${componentPromise})`;
-  const componentExport = `export const ${componentName} = ${componentLazy};`;
-
-  baklavaReactFileParts.push(componentExport);
+  const source = `
+  ${jsDoc}
+  export const ${componentName}: import("@lit-labs/react").ReactWebComponent<${componentType}> = (
+    () => createComponent<${componentType}>({
+      react: React,
+      displayName: ${componentName},
```

Comment:
I guess this should be a string?

Reply:
Yep, it's a string.

Reply:
I mean, it should be `displayName: "${componentName}",`, right?

Reply:
Yeah correct, missed it.